app/views/components/_rollbar.html.erb

```diff
@@ -0,0 +1,15 @@
+<script>
```

Comment:
I'm guessing this is for Rollbar client side JS?

Reply:
Yep